sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/ColumnVectorUtils.java

```diff
@@ -93,28 +93,6 @@ public static void populate(WritableColumnVector col, InternalRow row, int field
     }
   }
 
-  /**
-   * Returns the array data as the java primitive array.
-   * For example, an array of IntegerType will return an int[].
-   * Throws exceptions for unhandled schemas.
-   */
-  public static Object toPrimitiveJavaArray(ColumnarArray array) {
```

Comment:
Why this method? Looks it is just used in test.

Reply:
This method was used only for test. I removed this by replacing this with another method.